link/kprobe.go

```diff
@@ -245,6 +249,10 @@ func pmuProbe(typ probeType, args probeArgs) (*perfEvent, error) {
 
 	var config uint64
 	if args.ret {
+		// Use tracefs if we want to set kretprobe's maxactive.
+		if args.maxactive > 0 {
```

Comment:
This should be outside of the `args.ret` block, so that we refuse maxactive for regular kprobes as well.